src/vfs.js

```diff
@@ -27,6 +27,10 @@ fs.writeFileSync(
   "third_party/jquery-3.2.1.js",
   require("raw-loader!../third_party/jquery-3.2.1.js")
 );
+fs.writeFileSync(
+  "third_party/jshint.js",
+  require("raw-loader!../third_party/jquery-3.2.1.js")
```

Comment:
Is this (`jquery`) a copy-paste error?

Reply:
Yeah, looks like I must've overlooked that.